### PR TITLE
Adds Ship.IO UI-only example

### PR DESCRIPTION
**Pull request by charlesw-stripe**
## Motivation
Provide _UI only_ example of a Stripe app

### CODE REVIEW

**Review by ericfrank-stripe — COMMENTED** [review]
The code looks good to me—nice work! My team doesn't own this repo, so I'll let one of the other reviewers have the final say on merging.

File: examples/ship-io/src/hooks/LoginState.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { useContext, useState, createContext } from "react";
+import { LoginContext, ProviderProps } from "../common/types";
+import LoginView from "../views/LoginView";
+
+const LoginContext = createContext<LoginContext>({
+  isLoggedIn: false,
+  setIsLoggedIn: () => undefined,
+});
+
```
Comment (cching-stripe) [inline examples/ship-io/src/hooks/LoginState.tsx:9]: Nice job building the login page following the design standards!

File: examples/ship-io/src/views/LoginView.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          }}
+        >
+          <Box css={{ layout: "row", gap: 75, alignY: "center" }}>
+            <Inline>Sign in to Ship.io</Inline>
+            <Icon name="external" size="xsmall" />
+          </Box>
+        </Button>
+        <Inline>
+          Don&apos;t have an account?{" "}
+          <Link type="primary" href="#">
```
Comment (cching-stripe) [inline examples/ship-io/src/views/LoginView.tsx:66]: Should this redirect to Shipo site?

**Review by cching-stripe — COMMENTED** [review]
Looks great @charlesw-stripe ! Nice work. Added a few small NITs. Also would recommend adding in a small Readme to ensure devs know how to run the app and this is a sample.

Looks polished.

File: examples/ship-io/src/views/LoginView.tsx (hunk 2 above)
Comment (charlesw-stripe) [reply examples/ship-io/src/views/LoginView.tsx:66]: Our Ship.IO demo site isn't up. I'll update when it is.

**Review by cching-stripe — APPROVED** [review]
LGTM

**Pull request merged**